Implementations/Libuavcan/Platforms/index.md

```diff
@@ -25,23 +23,23 @@ Essentially, a libuavcan driver is a set of C++ classes implementing the followi
 * `uavcan::ICanIface`
 * `uavcan::ISystemClock`
 
-Listed interfaces are defined in
+The interfaces listed above are defined in
 [`libuavcan/include/uavcan/driver/`](https://github.com/UAVCAN/libuavcan/blob/master/libuavcan/include/uavcan/driver).
 
-Note that the library core already includes a non-blocking prioritized TX queue, so normally the driver should not
-implement software TX queue itself.
+Note that the library core already includes a non-blocking prioritized TX queue, so normally a driver should not
+implement a software TX queue itself.
 
 If the driver does not need to support redundant CAN bus interfaces,
 then the first two C++ interfaces can be implemented in the same C++ class.
 
 Some features can be left unimplemented:
 
-* IO flags - needed for dynamic node ID allocation and time synchronization master.
-* TX timestamping - needed for clock synchronization master.
-* Hardware CAN filters configuration -
+* **IO flags** - needed for dynamic node ID allocation and time synchronization master.
+* **TX timestamping** - needed for clock synchronization master.
+* **Hardware CAN filters configuration** -
 not necessary if there is enough computational power to delegate filtering to the library software.
 
-There are two polar examples of the driver complexity level that can give a basic idea of the development effort required:
+There are two extreme examples of the driver complexity level that can give a basic idea of the development effort required:
 
 * LPC11C24 driver - only necessary functionality, no redundant interface support, no TX timestamping - 600 LoC.
```

Comment:
Might be worth linking to the source code for both of these drivers from here. Then users won't have to search to see the complexity.